assets/src/edit-story/components/form/numeric.js

```diff
@@ -88,11 +90,23 @@ function Numeric({
         prefix={prefix}
         suffix={suffix}
         label={label}
-        value={`${value}${!focused ? symbol : ''}`}
+        value={
+          value === MULTIPLE_VALUE ? '' : `${value}${focused ? '' : symbol}`
+        }
         aria-label={ariaLabel}
         disabled={disabled}
         {...rest}
-        onChange={(evt) => onChange(evt.target.value, evt)}
+        onChange={(evt) => {
+          const newValue = evt.target.value;
```

Comment:
What if the value is temporarily empty for inserting a new value? Currently, it looks like it's not possible to empty a field.

Reply:
I suppose that's a general issue among the fields, not related to the goal of this PR specifically.

Reply:
Fixed.

Reply:
@dvoytenko I can still see the issue when testing, for example: selecting a text element, then clicking into Width and backspacing to delete all the numbers, the last number is still visible and the following is output to console:
<img width="671" alt="Screenshot 2020-03-23 at 11 02 16" src="https://user-images.githubusercontent.com/3294597/77305228-f582d180-6cf5-11ea-9d79-87b3292bd1f3.png">

Reply:
This has been fixed.

Reply:
Added test for this as well.